db_master_deploy/includes.sh

```diff
@@ -80,7 +95,7 @@ err() {
 #   integer
 #######################################
 Ceiling () {
```

Comment:
ceiling?

Reply:
yes it is for rounding up to the next heigher integer value